src/main/java/Duke.java

```diff
@@ -1,10 +1,160 @@
+import java.util.Scanner;
+
+/**
+ * Represents the bot Duke.
+ * Duke can perform tasks based on user input.
+ *
+ * {@value #NUM_OF_TASKS} Number of Tasks.
+ */
 public class Duke {
+
+    private static int NUM_OF_TASKS = 0;
+    private static Task[] list = new Task[100];
+
+    /**
+     * Runs other methods for main.
+     *
+     * @param args User Input.
+     */
     public static void main(String[] args) {
+        printGreetings();
+        listActions();
+    }
+
+    /**
+     * Prints greetings.
+     */
+    public static void printGreetings(){
```

Comment:
You may wanna set the access modifier to private to limit the scope of these methods if they are not gonna be used outside of the Duke class. Similarly for the other methods in Duke.